src/test/java/bio/terra/app/controller/DatasetsApiControllerTest.java

```diff
@@ -239,6 +245,61 @@ void testUpdateSnapshotBuilderSettings() throws Exception {
     verify(snapshotBuilderService).updateSnapshotBuilderSettings(DATASET_ID, SETTINGS);
   }
 
+  @Test
+  void testCreateCriteriaData() throws Exception {
+    TestUtils.mapFromJson(
```

Comment:
I realize the point here is that the json conversion doesn't throw, but I think a test should have at least one assert per conversion. Something like
```
    var criteria = TestUtils.mapFromJson(...);
    assertThat(critera.name(), is("name"));
```